pysat/_orbits.py

```diff
@@ -103,14 +103,14 @@ def __init__(self, sat=None, index=None, kind=None, period=None):
             raise ValueError('Unknown kind of orbit requested.')
 
         self._orbit_breaks = []
-        self.num = 0 #[]
+        self.num = 0  # []
```

Comment:
Shouldn't we just remove this old way of doing things?

Reply:
agreed